app/models/display_ad.rb

```diff
@@ -26,9 +26,14 @@ def process_markdown
     renderer = Redcarpet::Render::HTMLRouge.new(hard_wrap: true, filter_html: false)
     markdown = Redcarpet::Markdown.new(renderer)
     initial_html = markdown.render(body_markdown)
-    stripped_html = ActionController::Base.helpers.sanitize initial_html.html_safe,
-                                                            tags: %w[a em i b u br img h1 h2 h3 h4 div],
-                                                            attributes: %w[href target src height width style]
+
+    # Temporarily disable the sanitisation in order to launch the SheCoded Campaign.
```

Comment:
Should we add a note for team-cool to reenable this on a certain date? I've found that it's pretty easy for these kinds of things to be forgotten.

Even better: can we create an issue or a ticket to re-eenable, and then link to it here? We've done [this kind of thing](https://github.com/thepracticaldev/dev.to/search?q=delightful&unscoped_q=delightful) in team delightful, which has made it super easy for us to know which lines of code need our attention, because we can just `grep` for certain terms :)

Reply:
@vaidehijoshi yeah will definitely make a ticket, as not only does it need to re-enabled but it also needs to be fixed just for future ads. I'll create the ticket and hoping to tackle it during the cooldown.

Reply:
adding the team name is a good idea too, I'll pop them in the TODO in the codebase

Reply:
Cool. Is there a certain date that this needs to be reenabled by (i'm guessing whenever shecoded ends?). Might want to include that in the ticket and link to the ticket?

Reply:
@vaidehijoshi there's not a set date on when it needs to be enabled since it's not directly related to shecoded and it might affect future ads that need to use the css properties like these. I have created the ticket and assigned it to myself to add to my pipeline for the cooldown 🙂